packages/front-end/pages/settings/index.tsx

```diff
@@ -398,10 +398,8 @@ const GeneralSettingsPage = (): React.ReactElement => {
               </TempMessage>
             )}
           </div>
-          <div>
+          <div style={{ marginRight: "4rem" }}>
             <Button
-              style={{ marginRight: "4rem" }}
-              color={"primary"}
```

Comment:
The old button would show errors inline.  If we want to switch this to the new button, we should use the `setError` prop and manage state on this page.  Otherwise, errors while submitting will be swallowed.